src/pages/learn/index.jsx

```diff
@@ -8,32 +8,27 @@ let supportedEngines = [
   {
     id: 'godot',
     name: 'Godot',
-    href: '/learn/godot',
-    join: <span>,&nbsp;</span>
+    href: '/learn/godot/tutorials/crash-course'
```

Comment:
Why do these links take us to this: https://9f24440f.site-ckd.pages.dev/learn/godot/tutorials/crash-course
Instead of this: https://9f24440f.site-ckd.pages.dev/learn/godot

The latter seems like it should be the one that people land on from this page: https://9f24440f.site-ckd.pages.dev/learn